src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -429,6 +420,13 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     );
   }
 
+  private _measureCard() {
+    const card = this.shadowRoot!.querySelector("ha-card")!;
```

Comment:
The trick is here. In the Lovelace view each card is set to display: block so we can add a margin around them. This doesn't happen in editor/gallery. So we measure `<ha-card>`, which can be measured.

Reply:
Nice!